configure.ac

```diff
@@ -6,7 +6,9 @@ define(_CLIENT_VERSION_REVISION, 99)
 define(_CLIENT_VERSION_BUILD, 0)
 define(_CLIENT_VERSION_IS_RELEASE, false)
 define(_COPYRIGHT_YEAR, 2016)
-AC_INIT([Bitcoin Core],[_CLIENT_VERSION_MAJOR._CLIENT_VERSION_MINOR._CLIENT_VERSION_REVISION],[https://github.com/bitcoin/bitcoin/issues],[bitcoin])
+define(_COPYRIGHT_HOLDERS,[The %s developers])
```

Comment:
Moving it doesn't solve my issue; to be clear: having the copyright in a autoconf variable means it is passed to the compiler as a `-D...` flag. I think this is ridiculous. Copyright should be in the code, not something that can be varied by changing a compiler argument.

@jtimon's solution sounds somewhat better to me. It needs to be impossible to get rid of our copyright by just changing build metadata, without changing the actual code.